### PR TITLE
Add a complete example for Behat testing

### PR DESCRIPTION
**Pull request by anavarre**
This PR originates from a discussion with @ba66e77 in which he asked me to share my own attempt at documenting setting up Behat testing. It's by now means good to go and certainly overlaps parts of the existing documentation. Let's find out the parts we might want to reuse.

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -130,6 +130,69 @@ Note that Cucumber is simply a Ruby based BDD library, whereas Behat is a
 PHP based BDD library. Best practices for tests writing apply to both.
 * [The training wheels came off](http://aslakhellesoy.com/post/11055981222/the-training-wheels-came-off)
 
+### Complete Behat example
+
+#### Local setup
+
+BLT currently expects Google Chrome (more exactly, the `google-chrome` binary) to be available on the local setup. If you're using a Mac and Chrome is installed, you should be good to go. With Linux (e.g. in a VM or container), install either Google Chrome or Chromium (when/if https://github.com/acquia/blt/issues/2663 is fixed).
```
[the remark this answers is not in the document]
Comment (anavarre) [reply docs/testing.md]: Not using DrupalVM. Perhaps we should document somewhere that if you're NOT using DrupalVM you need this, then?

**Comment by danepowell** [comment]
I extracted what seemed most relevant here as #3424. If you think additional documentation updates are necessary please open a new issue. Thanks.

File: docs/testing.md [hunk 2]
```diff
@@ -130,6 +130,69 @@ Note that Cucumber is simply a Ruby based BDD library, whereas Behat is a
...
+
+Example for Debian/Ubuntu:
+
+```
+sudo apt-get install chromium-driver
+```
+
+#### Generate a `local.yml` file
```
[the remark this answers is not in the document]
Comment (brooke-heaton) [reply docs/testing.md]: I believe you must first run `tests:behat:init:config` in order to generate the boilerplate files. I think this has been overlooked above.

File: docs/testing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (brooke-heaton) [reply docs/testing.md]: @ba66e77 I believe you must first run `tests:behat:init:config` in order to generate the boilerplate files prior to running anything like `blt tests:behat` and for most people there will be some degree of reconfiguration for paths etc. If you have a multisite at this point you start screaming into a pillow and go for a walk.

File: docs/testing.md [hunk 3]
```diff
@@ -130,6 +130,69 @@ Note that Cucumber is simply a Ruby based BDD library, whereas Behat is a
...
+project:
+  machine_name: mysite
+  local:
+    protocol: http
+    hostname: '${project.machine_name}.local'
+```
+
+* The `${project.machine_name}` is a YAML variable. In the above example, `hostname` will render as `mysite.local`.
+
+Now, run `blt setup:behat --site=mysite.local` from the project root. If nothing happens, then check the file doesn't already exist at `/tests/behat/local.yml`.
```
[the remark this answers is not in the document]
Comment (brooke-heaton) [reply docs/testing.md]: @anavarre yes. In that case you would need to run `blt setup`.

**Pull request closed without merging**